functions/uninstall.go

```diff
@@ -14,12 +15,16 @@ import (
 	"github.com/gravitl/netclient/wireguard"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/models"
+	"golang.org/x/exp/slog"
 )
 
 // Uninstall - uninstalls networks from client
 func Uninstall() ([]error, error) {
 	allfaults := []error{}
 	var err error
+	if runtime.GOOS == "windows" {
+		slog.Error("Please uninstall windows using the add/remove program. https://docs.netmaker.io/netclient.html#uninstalling")
```

Comment:
If this is initiated from Gui, the user will probably not see this log.  Suggest you create an error using errors.New, log it and append to allfaults

Reply:
done